crml/staking/src/rewards/mod.rs

```diff
@@ -230,6 +278,23 @@ impl<T: Trait> Module<T> {
 		));
 		(*payouts).to_vec()
 	}
+
+	/// Return the number of reward payouts that need to be processed in the current block.
+	/// The result is dependant on the number of the current era's remained payouts and the number
+	/// of remained blocks before a new era.
+	fn calculate_payout_quota(remained_payouts: usize, remained_blocks: T::BlockNumber) -> usize {
+		let payout_split_threshold = T::PayoutSplitThreshold::get();
+		if remained_payouts <= payout_split_threshold || remained_blocks == Zero::zero() {
```

Comment:
```suggestion
		if remaining_payouts <= payout_split_threshold || remaining_blocks.is_zero() {
```
could also do remaning_blocks zero check first to avoid storage read of `payout_split_threshold`

Reply:
Done